repo/packages/K/kafka/0/config.json

```diff
@@ -1,6 +1,35 @@
 {
-  "type": "object",
-  "properties": {},
-  "additionalProperties": false,
-  "required": []
+    "type": "object",
+    "properties": {
```

Comment:
Could you plz, clarify what is the reason for "namespacing" them? We have only 6 properties, so there is no big reason to "structure" them. Beside that, if "namespaced", properties would be called like "kafka/image", "kafka/master", "kafka/api", etc, that, IMHO, would be more confusing to end user then is now.